rest/rest-resources/src/main/java/org/apache/brooklyn/rest/resources/BundleResource.java

```diff
@@ -107,6 +113,42 @@ protected ManagedBundle lookup(String symbolicName, String version) {
         }
         return b;
     }
+    
+
+    @Override
+    public List<TypeSummary> getTypes(String symbolicName, String version) {
+        ManagedBundle b = lookup(symbolicName, version);
+        return TypeTransformer.bundleDetails(brooklyn(), b, ui.getBaseUriBuilder(), mgmt()).getTypes();
```

Comment:
Don't we need an entitlement check on the bundle here?

Reply:
`lookup` does the bundle entitlement check doesn't it?

Reply:
Indeed, I did check that but forgot about it, ignore me then.